indico/web/client/js/react/components/ICSCalendarLink.jsx

```diff
@@ -82,90 +66,58 @@ export default function ICSCalendarLink({
+      dispatch({type: 'OPEN', key, source});
       const url = await fetchURL(extraParams, source);
       dispatch({type: 'LOADED', url});
     }
   };
 
-  let trigger;
-
-  if (options.length > 1) {
-    trigger = (
-      <Dropdown
-        styleName="height-full"
-        icon={null}
-        trigger={
-          renderButton ? (
-            renderButton(() => {}, {open: dropdownOpen || popupState.open})
-          ) : (
-            <Button icon size="small">
-              <Icon name="calendar alternate outline" />
-              <Icon name="caret down" />
-            </Button>
-          )
-        }
-        pointing={dropdownPosition}
-        onOpen={() => {
-          setDropdownOpen(true);
-        }}
-        onClose={() => {
-          setDropdownOpen(false);
-        }}
-        {...restProps}
-      >
-        <Dropdown.Menu>
-          <Dropdown.Header>
-            <Translate>Export</Translate>
-          </Dropdown.Header>
-          <Dropdown.Divider />
-          {options.map(({key, text, extraParams}) => (
-            <Dropdown.Item
-              key={key}
-              text={text}
-              onClick={() => handleSetOption(text, extraParams)}
-            />
-          ))}
-        </Dropdown.Menu>
-      </Dropdown>
-    );
-  } else {
-    const {text, extraParams} = options[0];
-    trigger = renderButton ? (
-      renderButton(() => handleSetOption(text, extraParams), {open: popupState.open})
-    ) : (
-      <Button icon size="small" onClick={() => handleSetOption(text, extraParams)}>
-        <Icon name="calendar alternate outline" />
-        <Icon name="caret down" />
-      </Button>
-    );
-  }
-
   return (
     <Popup
-      trigger={trigger}
+      trigger={
+        renderButton ? (
+          renderButton({open: popupState.open})
+        ) : (
+          <Button icon size="small">
+            <Icon name="calendar alternate outline" />
+            <Icon name="caret down" />
+          </Button>
+        )
+      }
       position={popupPosition}
       open={popupState.open}
       popperDependencies={[copied]}
       on="click"
-      onOpen={() => {
+      onOpen={async () => {
+        const {key, extraParams} = options[0];
+        await handleSetOption(key, extraParams);
         setCopied(false);
       }}
       onClose={() => {
         handleClose();
       }}
       wide
     >
-      <Header
-        styleName="export-header"
-        content={Translate.string('Export')}
-        subheader={<Label color="blue">{popupState.text}</Label>}
-      />
+      <Header styleName="export-header">
+        <Button.Group size="small">
+          {options.map(({key, text, extraParams}, idx) => (
+            <>
+              <Button
+                key={key}
+                content={text}
+                onClick={() => handleSetOption(key, extraParams)}
+                primary={key === popupState.key}
+                label={idx === 0 ? Translate.string('Export') : undefined}
```

Comment:
can we render the label separately somehow? it's a bit weird that it acts as a label for the first actual button (and is clickable / has the cursor associated with clickable elements)

if it's not possible to render the label separately, maybe some css to disable pointer-events on it would help...

Reply:
ok looks like disabling pointer events doesn't work because of the DOM structure used for those labels..

Reply:
Yes, I noticed that too, seems like `cursor: initial` in the wrapper `ui left labeled button` works, the button preserves the click.